pkg/block/azure/adapter.go

```diff
@@ -139,6 +139,7 @@ func ResolveBlobURLInfoFromURL(pathURL *url.URL) (BlobURLInfo, error) {
 
 func resolveBlobURLInfo(obj block.ObjectPointer) (BlobURLInfo, error) {
 	key := obj.Identifier
+	// TODO (gilo): verify there's no need for StorageID here
```

Comment:
At this point you are already in the context of a specific adapter instance so storage ID is not needed here